front-end-admin/src/components/user/UserList.vue

```diff
@@ -64,15 +65,16 @@ export default {
   },
   methods: {
     getPermissions(user) {
-      if (user.permissions === null || user.permissions === 'BASIC_USER') {
-        return 'No Permissions';
-      } else if (user.permissions === 'ADMINISTRATOR') {
-        return 'Admin';
-      } else if (user.permissions === 'SUPERADMIN') {
-        return 'Jody/SuperAdmin';
-      }
-      user.permissions = 'BASIC_USER';
-      return 'No Permissions';
+      // if (user.permissions === null || user.permissions === 'BASIC_USER') {
+      //   return 'No Permissions';
+      // } else if (user.permissions === 'ADMINISTRATOR') {
+      //   return 'Admin';
+      // } else if (user.permissions === 'SUPERADMIN') {
+      //   return 'Jody/SuperAdmin';
+      // }
+      // user.permissions = 'BASIC_USER';
+      // return 'No Permissions';
```

Comment:
you're killing me

Reply:
didn't realize satan was making code contributions to this project

Reply:
// I'M DYING INSIDE

Reply:
Don't burn the ships...